### ISSUE TITLE
"Upgrade a Helm Chart" step only allows Files from Additional Packages to be downloaded directly to the execution target (UI Bug)

### ISSUE THREAD

**Issue by danefalvo** [OctopusDeploy/Issues#7509]
### Team

- [X] I've assigned a team label to this issue

### Severity

some users inconvenienced - sub-par Workaround available

### Version

2022.1.2412

### Latest Version

I could reproduce the problem in the latest build

### What happened?

When using the step "Upgrade a Helm Chart" if you select **"Files in Additional Packages"**
then specify a package from an external feed, you are unable to alter the acquisition location via the UI.

The second option "The Package will be downloaded directly from the feed on the execution target" will always be selected unless over-ridden by a bound value. 

In other words, "AcquisitionLocation" will always be set to "ExecutionTarget".

### Reproduction

https://user-images.githubusercontent.com/5361797/165094459-85f25619-563c-440e-be59-feae12c002ed.mov

### More Information

Originally raised here: https://github.com/OctopusDeploy/Issues/issues/7326
